src/utilities/codemirror/codemirror-utils.ts

```diff
@@ -120,10 +120,10 @@ export function getDefaultVariableArgs(parameters: VariableDeclaration[]): strin
         return parameter.allowable_values && parameter.allowable_values.length > 0
           ? `"${parameter.allowable_values[0]}"`
           : parameter.enum_name
-            ? `${parameter.enum_name}`
+            ? `"${parameter.enum_name}"`
```

Comment:
Can we update the test for this case?

Reply:
Sure added a unit test to test this path